package.json

```diff
@@ -48,43 +49,43 @@
 		"migration-prune": "migrate prune --autosync --config ./config/migrate.js",
 		"migration-persist": "npm run nest:start:console -- database export --collection migrations --override",
 		"migrate-etherpads": "node ./migrate-etherpads.js",
-		"nest:prebuild": "rimraf dist",
-		"nest:build": "nest build",
+		"nest:prebuild": "rimraf dist & npm run ignore-feathers-error",
+		"nest:build": "npm run nest:prebuild & nest build",
```

Comment:
to note, the prebuild was not called automatically. So every time, the dist folder is not removed. 
Probably nest takes care of that automatically quite well. 
I am not sure in the end if is safer to remove it.
According to npm, scripts with suffix pre or post are run automatically before / after the script. This does not work in our case, becasue we start with nest ... a better waybto restructure would be
nest:start:dev  => start:nest:dev, whoch in theory supports prefixing i.e. prestart:nest:dev